readability - tutorials/howto (#188)
unfoldtoolbox/Unfold.jl: docs/src/tutorials/lm_mu.md

nothing # hide
```
## Inspection

> The data has some noise. The underlying signal pattern is a positive-negative-positive spike.

Reviewer: ```suggestion
The data has only little noise. The underlying signal pattern is a positive-negative-positive spike.
```

```@example Main
times = range(1/50, length=200, step=1/50)
f = Figure()
plot(f[1, 1], data[1:200], times)
vlines!(evts[evts.latency .<= 200, :latency] ./ 50) # show events, latency in samples!
f
```

To inspect the event dataframe we use
```@example Main
show(first(evts, 6), allcols = true)
```

> Every row is an experimental event. Note that `:latency` is meausred as the sample timestamps, whereas `:onset` would typically refer to seconds.

Reviewer: ```suggestion
Every row is an experimental event. Note that `:latency` refers to time in samples, whereas `:onset` would typically refer to seconds.
```

## Traditional Mass Univariate Analysis
To perform a mass univariate analysis, you must complete the following steps:

1. Split data into epochs 
2. Specify a formula 
3. Fit a linear model to each time point & channel
4. Visualize the results.

#### 1. Split data into epochs 
Initially, you have data with a duration that represents the whole experimental trial. You need to cut the data into small regular epochs related to the some event, e.g. start of fixation.

```@example Main
# we have multi channel support
data_r = reshape(data, (1,:))
# cut the data into epochs
data_epochs, times = Unfold.epoch(data = data_r, tbl = evts, τ = (-0.4, 0.8), sfreq = 50);
size(data_epochs)
```
- `τ` specifies the epoch size.
- `sfreq` - sampling rate, converts `τ` to samples.

```@example Main
typeof(data_epochs)
```
!!! note

>    Partial trials could be included or ignored by the corresponding functions due to `missing` data type. Check functions such as the Julia-based `disallowmissing` and the internal `Unfold.dropMissingEpochs`.

Reviewer: ```suggestion
   Partial trials could be included or ignored by the corresponding functions due to `missing` data type. Check functions such as the Julia-based `disallowmissing` and the internal `Unfold.drop_missing_epochs`.
```
